conans/client/conan_api.py

```diff
@@ -556,14 +556,11 @@ def config_rm(self, item):
         self._client_cache.invalidate()
 
     @api_method
-    def config_install(self, item, verify_ssl, config_type=None, args=None):
-        # _make_abs_path, but could be not a path at all
-        if item is not None and os.path.exists(item) and not os.path.isabs(item):
-            item = os.path.abspath(item)
```

Comment:
This is removed, but seems it was in place for something. Maybe it is not covered by tests, but please check why it was removed.

Reply:
This has been moved here: https://github.com/conan-io/conan/pull/4083/files#diff-32564897949777992206da1e43543b08R127 becouse the item is no longer a path or url, but a structure with type of config, address (path or url) and arguments. I am sure it is covered by the tests below